nodes/src/nodes/search_exa/IGlobal.py

```
@@ -43,7 +43,7 @@ def _get_apikey(cfg: dict, conn_config: dict) -> str:
         apikey = str((conn_config.get('apikey') or '')).strip()
         if apikey:
             return apikey
-        return str((os.environ.get('ROCKETRIDE_APIKEY_EXA') or '')).strip()
+        return str((os.environ.get('ROCKETRIDE_EXA_KEY') or '')).strip()
```

Ruff (0.15.10)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)